temporalio/client.py

```diff
@@ -71,16 +71,16 @@ async def connect(
         default_workflow_query_reject_condition: Optional[
             temporalio.common.QueryRejectCondition
         ] = None,
-        tls_config: Optional[TLSConfig] = None,
+        tls: Union[bool, TLSConfig] = False,
```

Comment:
I'd consider accepting `None` here too

Reply:
That seems ambiguous. We used to accept `None` here when this wasn't how you chose whether TLS is used, but now a `tls=False` is clearer. The parameter changed from "tls configuration" to "tls enabled, with configuration if necessary".